hosted/json-schemas/offering.schema.json

```diff
@@ -1,82 +1,152 @@
+              },
+              "min": {
+                "$ref": "definitions.json#/definitions/decimalString",
+                "description": "minimum amount required to use this payment method."
+              },
+              "max": {
+                "$ref": "definitions.json#/definitions/decimalString",
+                "description": "maximum amount allowed when using this payment method."
+              },
+              "fee": {
+                "$ref": "definitions.json#/definitions/decimalString",
+                "description": "Fee charged to use this payment method. absence of this field implies that there is no _additional_ fee associated to the respective payment method"
+              }
+            },
+            "required": ["kind"]
+          }
         }
       },
-      "required": ["currencyCode"]
+      "required": ["currencyCode", "methods"]
     },
-    "PaymentMethod": {
+    "payout": {
       "type": "object",
       "additionalProperties": false,
       "properties": {
-        "kind": {
+        "currencyCode": {
           "type": "string",
-          "description": "The type of payment method. e.g. BITCOIN_ADDRESS, DEBIT_CARD, etc."
+          "description": "ISO 3166 currency code string"
+        },
+        "min": {
+          "$ref": "definitions.json#/definitions/decimalString",
+          "description": "Minimum amount of currency that can be requested"
+        },
+        "max": {
+          "$ref": "definitions.json#/definitions/decimalString",
+          "description": "Maximum amount of currency that can be requested"
         },
-        "requiredPaymentDetails": {
-          "type": "object",
-          "description": "A JSON Schema containing the fields that need to be collected in order to use this payment method"
+        "methods": {
+          "type": "array",
+          "items": {
+            "type": "object",
+            "additionalProperties": false,
+            "properties": {
+              "kind": {
+                "type": "string",
+                "description": "The type of payment method. e.g. BITCOIN_ADDRESS, DEBIT_CARD, etc."
+              },
+              "name": {
+                "type": "string",
+                "description": "Payment Method name. Expected to be rendered on screen."
+              },
+              "description": {
+                "type": "string",
+                "description": "Blurb containing helpful information about the payment method. Expected to be rendered on screen. e.g. \"segwit addresses only\""
+              },
+              "group": {
+                "type": "string",
+                "description": "value that can be used to group specific payment methods together e.g. Mobile Money vs. Direct Bank Deposit"
+              },
+              "requiredPaymentDetails": {
+                "type": "object",
+                "description": "A JSON Schema containing the fields that need to be collected in order to use this payment method"
```

Comment:
is there a JSON schema we can add a `$ref` to?

Reply:
This is intended to be a json schema for which the PFI can define for their given payment use case; as in, it's a generalized space for a PFI to define payment details via JSON Schema

Reply:
we should add something like this to restrict the type https://github.com/w3c/vc-json-schema/blob/main/schema/json-schema-credential-schema.json#L21

Reply:
oh nice, interesting... yeah I guess that makes sense, a json schema is itself a type

Reply:
nice!!

Reply:
done. good call @decentralgabe. added you as commit co-author: https://github.com/TBD54566975/tbdex/commit/efd97df9f6feed4ef58a5257dad7a3c5d26a9bd9